source/quick-start.txt

```diff
@@ -28,74 +28,99 @@ instance.
 Set up Your Project
 -------------------
 
-Install Node and NPM
+Install Node and npm
 ~~~~~~~~~~~~~~~~~~~~
 
-Ensure your system has Node.js version 10 or later and a compatible
-version of NPM (Node Package Manager) installed. For more information on how
-to check your version of Node and NPM and installation instructions for your
-system, see `downloading and installing Node.js and npm
+Ensure your system has Node.js version 12 or later and a compatible
+version of npm (Node Package Manager) installed. For information on how
+to check your version of Node and npm, as well as installation
+instructions, see `downloading and installing Node.js and npm 
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
-Initialize the Project
-~~~~~~~~~~~~~~~~~~~~~~
+.. tabs::
 
-After you verify that you installed the required versions of Node and NPM,
-create a new project.
+   .. tab:: Clone the Project
+      :tabid: clone the project
 
-First, create a directory for your project in your command line interface:
+      First, install Git using `Git's Getting Started Guide
+      <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__.
 
-.. code-block:: none
+      Then, in your shell, clone the `js-starter repository
+      <https://github.com/mongodb-university/js-starter>`__:
 
-   mkdir node_quickstart
+      .. code-block:: bash
 
-Then, navigate into that directory so you can work directly with your
-project's files:
+         git clone https://github.com/mongodb-university/js-starter.git
 
-.. code-block:: none
+      Next, navigate into the repository:
 
-   cd node_quickstart
+      .. code-block:: bash
 
-Next, set up NPM for your project by running the following command.
-This command creates a file called ``package.json``:
+         cd js-starter
 
-.. code-block:: none
+      Then, install the Node.js driver:
 
-   npm init -y
+      .. code-block:: none
 
-.. note:: Why the -y?
+         npm install
 
-   If you specify the ``-y`` option in the command, NPM automatically
-   accepts the default values for the command. Omit the ``-y`` flag to
-   interactively select your project settings.
+      This command performs the following actions:
 
-Add MongoDB as a Dependency
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      - Downloads and saves the ``mongodb`` package in a directory
+        called ``node_modules``
+      - Installs ``eslint``, which the project uses to identify and
+        report issues in your code
+      - Installs ``prettier``, which the project uses to format your code uniformly
```

Comment:
Nit: I'm not sure what is meant by "format uniformly" (maybe "format consistently"?), I think just "format your code" sounds more correct to my ear, as I think it implies consistent formatting.

```suggestion
      - Installs ``prettier``, which the project uses to format your code
```